.github/workflows/build.yml

```diff
@@ -0,0 +1,90 @@
+on:
+  - pull_request
+  - push
+
+name: build
+
+jobs:
+  tests:
+    name: PHP ${{ matrix.php }}-${{ matrix.os }}-${{ matrix.coverage }}
+
+    env:
+      extensions: curl, mbstring, openssl, pdo, pdo_sqlite
+      SYMFONY_DEPRECATIONS_HELPER: weak
+
+    runs-on: ${{ matrix.os }}
+
+    continue-on-error: ${{ matrix.experimental }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
Can it run on Windows? If so, `appveyor.yml` could be removed.

Reply:
done.